aptos-move/e2e-move-tests/Cargo.toml

```diff
@@ -22,7 +22,7 @@ aptos-gas-profiling = { workspace = true }
 aptos-gas-schedule = { workspace = true, features = ["testing"] }
 aptos-language-e2e-tests = { workspace = true }
 aptos-package-builder = { workspace = true }
-aptos-types = { workspace = true }
+aptos-types = { workspace = true, features = ["testing"] }
```

Comment:
Do we really need testing features in production code?

The "testing" feature requirement should be moved to the dev-dependencies section.